das/checkpoint.go

```diff
@@ -5,18 +5,18 @@ import (
 )
 
 type checkpoint struct {
-	SampleFrom  uint64 `json:"sample_from"`
-	NetworkHead uint64 `json:"network_head"`
+	SampleFrom  uint `json:"sample_from"`
```

Comment:
We don't see the added benefit of this change - it really only bloats the diff for no improvement. Also @walldiss made a point that if we decide to use atomics later, we'd need to revert anyway. Can you please revert this and use `uint64` in the config

Reply:
For clarity - we're referring to all `uint64` -> `uint` changes

Reply:
Claro 👍🏻